examples/sample-foundry-project/cannonfile.old.toml

```diff
@@ -3,15 +3,16 @@ version = "<%= package.version %>"
 description = "Simple project to verify the functionality of cannon"
 keywords = ["sample", "greeter"]
 
-[setting.value]
```

Comment:
this file is specifically designed to test the behavior of the old `setting`s format and other things before those renaming changes, so changing from `setting` to `var` here is not a good idea